pysma/const.py

```diff
@@ -229,65 +235,400 @@
+}
+SENSOR_BATTERY_AC_POWER_DISCHARGE_A = { # Example: "6100_00499400":{"7":[{"val":0},{"val":null},{"val":null}]}
+    SKEY: "6100_00499400_0",
+    SNAME: "battery_ac_power_discharge_a",
+    SUNIT: "W",
+}
+SENSOR_BATTERY_AC_POWER_DISCHARGE_B = { # Example: "6100_00499400":{"7":[{"val":0},{"val":null},{"val":null}]}
+    SKEY: "6100_00499400_1",
+    SNAME: "battery_ac_power_discharge_b",
+    SUNIT: "W",
+}
+SENSOR_BATTERY_AC_POWER_DISCHARGE_C = { # Example: "6100_00499400":{"7":[{"val":0},{"val":null},{"val":null}]}
+    SKEY: "6100_00499400_2",
+    SNAME: "battery_ac_power_discharge_c",
+    SUNIT: "W",
+}
+SENSOR_BATTERY_AC_REACTIVE_POWER = { # Example: "6100_40265F00":{"7":[{"val":-3}]}
+    SKEY: "6100_40265F00",
+    SNAME: "battery_ac_reactive_power",
+    SUNIT: "var",
+}
+SENSOR_BATTERY_AC_REACTIVE_POWER_L1 = { # Example: "6100_40666000":{"7":[{"val":-6}]} # 10044=GridMs.VAr.phsA\6709248\Reactive power L1
+    SKEY: "6100_40666000",
+    SNAME: "battery_ac_reactive_power_l1",
+    SUNIT: "var",
+}
+SENSOR_BATTERY_AC_REACTIVE_POWER_L2 = { # Example: "6100_40666100":{"7":[{"val":null}]} # 10045=GridMs.VAr.phsB\6709504\Reactive power L2
+    SKEY: "6100_40666100",
+    SNAME: "battery_ac_reactive_power_l2",
+    SUNIT: "var",
+}
+SENSOR_BATTERY_AC_REACTIVE_POWER_L3 = { # Example: "6100_40666200":{"7":[{"val":null}]} # 10046=GridMs.VAr.phsC\6709760\Reactive power L3
+    SKEY: "6100_40666200",
+    SNAME: "battery_ac_reactive_power_l3",
+    SUNIT: "var",
+}
+SENSOR_BATTERY_AC_APPARENT_POWER = { # Example: "6100_40666700":{"7":[{"val":0}]} # 10047=GridMs.TotVA\6711040\Apparent power
+    SKEY: "6100_40666700",
+    SNAME: "battery_ac_apparent_power",
+    SUNIT: "VA",
+}
+SENSOR_BATTERY_AC_APPARENT_POWER_L1 = { # Example: "6100_40666800":{"7":[{"val":0}]} # 10048=GridMs.VA.phsA\6711296\Apparent power L1
+    SKEY: "6100_40666800",
+    SNAME: "battery_ac_apparent_power_l1",
+    SUNIT: "VA",
+}
+SENSOR_BATTERY_AC_APPARENT_POWER_L2 = { # Example: "6100_40666900":{"7":[{"val":null}]} # 10049=GridMs.VA.phsB\6711552\Apparent power L2
+    SKEY: "6100_40666900",
+    SNAME: "battery_ac_apparent_power_l2",
+    SUNIT: "VA",
+}
+SENSOR_BATTERY_AC_APPARENT_POWER_L3 = { # Example: "6100_40666A00":{"7":[{"val":null}]} # 10050=GridMs.VA.phsC\6711808\Apparent power L3
+    SKEY: "6100_40666A00",
+    SNAME: "battery_ac_apparent_power_l3",
+    SUNIT: "VA",
+}
+SENSOR_BATTERY_AC_POWER_FACTOR = { # Example: "6100_00665900":{"7":[{"val":999}]} # 10710=GridMs.TotPFPrc\6707456\Displacement power factor
+    SKEY: "6100_00665900",
+    SNAME: "battery_ac_power_factor",
+    SUNIT: "",
+    SFACTOR: 1000,
+}
+SENSOR_BATTERY_AC_POWER_FACTOR_EXCITATION = { # Example: "6180_08465A00":{"7":[{"val":[{"tag":1042}]}]} # 10711=GridMs.TotPFExt\4610560\Excitation type of cos φ # 1041=OvExt\0\Overexcited # 1042=UnExt\0\Underexcited
+    SKEY: "6180_08465A00",
+    SNAME: "battery_ac_power_factor_excitation",
+    SPATH: JMESPATHS_TAG,
+    SL10N: True,
+}
```

Comment:
These are all related to grid instead of battery? A name like `SENSOR_GRID_REACTIVE_POWER_L1` makes it more generic.